.eslintrc.cjs

```diff
@@ -4,5 +4,6 @@ module.exports = {
     'tsdoc/syntax': 'warn',
     'max-len': ['error', 200],
     camelcase: 'off',
+    '@typescript-eslint/no-unused-vars': 'warn',
```

Comment:
Can you use error instead of warn and fix problems if you see some when running `npm run fmt:check`?